code/modules/hydroponics/grown/garlic.dm

```diff
@@ -20,3 +21,47 @@
 	bitesize_mod = 2
 	tastes = list("garlic" = 1)
 	wine_power = 10
+
+/obj/item/seeds/garlic/hairy
+	name = "pack of hair garlic"
+	desc = "A packet of extremely hairy seeds."
+	icon_state = "seed-garlic"
+	species = "garlic"
+	plantname = "Garlic Sprouts"
+	product = /obj/item/reagent_containers/food/snacks/grown/garlic/hairy
+	yield = 6
+	potency = 25
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	mutatelist = list(/obj/item/seeds/garlic/hairy)
+	reagents_add = list(/datum/reagent/consumable/garlic = 0.15, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/reagent_containers/food/snacks/grown/garlic/hairy
+	seed = /obj/item/seeds/garlic/hairy
+	name = "hairy garlic"
+	desc = "Delicious, but with a potentially overwhelming odor AND disgusting hair on it like you dropped it on the floor and didn't meet the 5 second rule in time."
```

Comment:
add a comma after "it"? or is how it is currently grammatically correct?

Reply:
maybe runon